code/controllers/subsystem/achievements.dm

```diff
@@ -47,6 +48,23 @@ SUBSYSTEM_DEF(achievements)
 	qdel(ridOldChieves)
 	return ..()
 
+/datum/controller/subsystem/achievements/fire(resumed)
+	//The solar panel achievement
+	if(!CE)
```

Comment:
This whole thing doesn't account for there being multiple CE's :)

Reply:
Does it *really* need to?

Reply:
Yes

Reply:
No it doesnt, codebase does not give support for many heads and it is impossible unless manipulation of job subsystem to have a higher limit. This is for assigned_role, aka round start/latejoin CEs which only one can normally spawn and HOP is unable to open more slots

Reply:
Look at the alexkar review note below.

Reply:
Nothing is impossible, you should assume there are always more than one of everything.

Reply:
Yogs isn't exactly known for their defensive programming

Reply:
or for any programming at all tbh

Reply:
guys don't be rude